src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,19 @@
+.footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  background-color: var(--footer-background);
+  height: var(--footer-height);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.wrapper {
+  padding: 0 24px;
+
+  a {
```

Comment:
a little bit too generic name. maybe check something like this:
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css